app/controllers/admins_controller.rb

```diff
@@ -475,6 +475,8 @@ def detectAndConvertRoster(roster)
       # First Name(6), (skip)(7), Andrew ID(8), (skip)(9), School(10), 
       # Major(11), Year(12), (skip)(13), Grade Policy(14), ...
 
+      # Sanitize roster input, ignoring empty / incomplete lines.
+      parsedRoster.select! { |row| row.length == CMU_ROSTER_COLUMNS }
```

Comment:
@dlbucci I changed `CMU_ROSTER_COLUMNS` to `NEW_ROSTER_COLUMNS`.

However, we still need to check `length == NEW_ROSTER_COLUMNS` twice because the first time is when we distinguish if the roster is of new format or old format; the second time is when I filter for incomplete rows, assuming the roster is of new format.